cli/exec/flags.go

```diff
@@ -125,229 +125,277 @@ var flags = []cli.Flag{
 		Name:    "repo-url",
+		Usage:   "Set the metadata environment variable \"CI_REPO_URL\".",
 	},
 	&cli.StringFlag{
 		Sources: cli.EnvVars("CI_REPO_SCM"),
 		Name:    "repo-scm",
+		Usage:   "Set the metadata environment variable \"CI_REPO_SCM\".",
 		Value:   "git",
 	},
 	&cli.StringFlag{
 		Sources: cli.EnvVars("CI_REPO_DEFAULT_BRANCH"),
 		Name:    "repo-default-branch",
+		Usage:   "Set the metadata environment variable \"CI_REPO_DEFAULT_BRANCH\".",
 		Value:   "main",
 	},
 	&cli.StringFlag{
 		Sources: cli.EnvVars("CI_REPO_CLONE_URL"),
 		Name:    "repo-clone-url",
+		Usage:   "Set the metadata environment variable \"CI_REPO_CLONE_URL\".",
 	},
 	&cli.StringFlag{
 		Sources: cli.EnvVars("CI_REPO_CLONE_SSH_URL"),
 		Name:    "repo-clone-ssh-url",
+		Usage:   "Set the metadata environment variable \"CI_REPO_CLONE_SSH_URL\".",
 	},
 	&cli.StringFlag{
 		Sources: cli.EnvVars("CI_REPO_PRIVATE"),
 		Name:    "repo-private",
+		Usage:   "Set the metadata environment variable \"CI_REPO_PRIVATE\".",
 	},
 	&cli.BoolFlag{
 		Sources: cli.EnvVars("CI_REPO_TRUSTED"),
 		Name:    "repo-trusted",
+		Usage:   "Set the metadata environment variable \"CI_REPO_TRUSTED\".",
 	},
 	&cli.IntFlag{
 		Sources: cli.EnvVars("CI_PIPELINE_NUMBER"),
 		Name:    "pipeline-number",
+		Usage:   "Set the metadata environment variable \"CI_PIPELINE_NUMBER\".",
 	},
 	&cli.IntFlag{
 		Sources: cli.EnvVars("CI_PIPELINE_PARENT"),
 		Name:    "pipeline-parent",
+		Usage:   "Set the metadata environment variable \"CI_PIPELINE_PARENT\".",
 	},
 	&cli.IntFlag{
 		Sources: cli.EnvVars("CI_PIPELINE_CREATED"),
 		Name:    "pipeline-created",
+		Usage:   "Set the metadata environment variable \"CI_PIPELINE_CREATED\".",
 	},
 	&cli.IntFlag{
 		Sources: cli.EnvVars("CI_PIPELINE_STARTED"),
 		Name:    "pipeline-started",
+		Usage:   "Set the metadata environment variable \"CI_PIPELINE_STARTED\".",
 	},
 	&cli.StringFlag{
 		Sources: cli.EnvVars("CI_PIPELINE_EVENT"),
 		Name:    "pipeline-event",
+		Usage:   "Set the metadata environment variable \"CI_PIPELINE_EVENT\".",
 		Value:   "manual",
 	},
 	&cli.StringFlag{
 		Sources: cli.EnvVars("CI_PIPELINE_FORGE_URL"),
 		Name:    "pipeline-url",
+		Usage:   "Set the metadata environment variable \"CI_PIPELINE_FORGE_URL\".",
 	},
 	&cli.StringFlag{
 		Sources: cli.EnvVars("CI_PIPELINE_DEPLOY_TARGET"),
 		Name:    "pipeline-deploy-to",
+		Usage:   "Set the metadata environment variable \"CI_PIPELINE_DEPLOY_TARGET\".",
 	},
 	&cli.StringFlag{
 		Sources: cli.EnvVars("CI_PIPELINE_DEPLOY_TASK"),
 		Name:    "pipeline-deploy-task",
+		Usage:   "Set the metadata environment variable \"CI_PIPELINE_DEPLOY_TASK\".",
 	},
 	&cli.StringFlag{
 		Sources: cli.EnvVars("CI_PIPELINE_FILES"),
-		Usage:   "either json formatted list of strings, or comma separated string list",
+		Usage:   "Set the metadata environment variable \"CI_PIPELINE_FILES\", either json formatted list of strings, or comma separated string list.",
 		Name:    "pipeline-files",
```

Comment:
```suggestion
		Name:    "changed-files",
```

Reply:
well it is an breaking change then :/

and it would break the name convention. what could work is:
```diff
-		Name:    "pipeline-files",
+
```		Name:    "pipeline-changed-files",

Reply:
well at least it is non breaking as not jet released ... https://github.com/woodpecker-ci/woodpecker/pull/4119

Reply:
`pipeline-changed-files` seems good

Reply:
->  #4142